### PR TITLE
Update front page

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by andf-viam — COMMENTED** [review]
This looks great! Only two comments, both about the carousel:
- Carousel right arrow now aligns to right edge of visible window, rather than remaining attached to right edge of carousel itself. Visible only when at full width.
- Carousel left and right arrows no longer seem to work.

Other than that, this is a fantastic improvement! I think the new card sizing, padding, and auto-collapsing works really well!

**Comment by npentrel** [comment]
I've fixed the carousel to only show arrows if not all elements are visible

File: docs/_index.md [hunk 1]
```diff
@@ -107,53 +111,109 @@ Viam is a complete software platform for robots that runs on Linux and macOS and
...
             <p style="text-align: left;">Have questions, or want to meet other people working on robots? <a href="https://discord.gg/viam">Join us in the Community Discord!</a></p>
+            </div>
             {{<gif webm_src="img/heart.webm" mp4_src="img/heart.mp4" alt="A robot drawing a heart">}}
         </div>
     </div>
     </div>
 </div>
 
+<br>
+<h2>Explore capabilities and APIs</h2>
```
Comment (npentrel) [inline docs/_index.md:173]:
```suggestion
<h2>Capabilities and APIs</h2>
```

**Comment by npentrel** [comment]
I've now also added some more content which I think makes the page a bit more well rounded - please re-review @andf-viam

**Review by andf-viam — APPROVED** [review]
Looks great! Carousel change 👍  and the new `Capabilities and APIs` sections looks enormously helpful - all our docs offerings at a glance.

Thanks!

**Pull request merged**